### PR TITLE
style: fixed style issues

### PR DESCRIPTION
**Pull request by nesl247**

### CODE REVIEW

File: src/Decorator/EchoDecorator.php [hunk 1]
```diff
@@ -8,22 +8,13 @@ class EchoDecorator extends SafeEchoDecorator
...
         return $openString;
     }
 
     /**
-     * @param string $string
-     * @param string $hiddenChar
-     *
      * @return string
      */
     public function hide(string $string, string $hiddenChar = '*')
```
Comment (alexandrecruz) [inline src/Decorator/EchoDecorator.php:20]: MIssing `:string` after declaration. docblock is not required here anymore also.

File: src/Decorator/SafeEchoDecorator.php [hunk 2]
```diff
@@ -9,18 +9,11 @@
...
      */
     abstract public function wrap(string $openString, string $hiddenString, $data = null): string;
 
     /**
-     * @param string $string
-     * @param string $hiddenChar
-     *
      * @return string
      */
     public function hide(string $string, string $hiddenChar = '*')
```
Comment (alexandrecruz) [inline src/Decorator/SafeEchoDecorator.php:19]: Also here.

File: src/Factory/DecoratorFactory.php [hunk 3]
```diff
@@ -39,8 +35,6 @@ public static function create($decoratorConfiguration): SafeEchoDecorator
     }
 
     /**
-     * @param string $decoratorClassName
-     *
      * @return SafeEchoDecorator|null
      */
     private static function createFromString(string $decoratorClassName)
```
Comment (alexandrecruz) [inline src/Factory/DecoratorFactory.php:40]: maybe `?SafeEchoDecorator`

File: src/Decorator/EchoDecorator.php (hunk 1 above)
Comment (nesl247) [reply src/Decorator/EchoDecorator.php:20]: This is simply running php-cs-fixer. I am not making any changes to the methods' signatures. Doing so requires investigating whether or not it returns a nullable type. Most docblocks are inaccurate.

**Pull request merged**